tests/feature_toggle/test_feature_toggle.py

```diff
@@ -0,0 +1,31 @@
+from mock import patch, Mock
+from parameterized import parameterized, param
+from unittest import TestCase
+
+from samtranslator.feature_toggle.feature_toggle import FeatureToggle, FeatureToggleLocalConfigProvider
+
+
+class TestFeatureToggle(TestCase):
+    @parameterized.expand(
+        [
+            param("feature-1", "beta", "default", False),
+            param("feature-1", "beta", "us-west-2", True),
+            param("feature-2", "beta", "us-west-2", False),  # because feature is missing
```

Comment:
We should test that the warn log message appears in this case - it's an important part of bad configuration detection.

Reply:
Added additional tests for warning code path from other config providers.